components/Tabs/src/TabBarView/private/MDCTabBarViewItemView.m

```diff
@@ -132,4 +134,14 @@ - (CGSize)sizeThatFits:(CGSize)size {
   return CGSizeMake(width, height);
 }
 
+#pragma mark - Accessibility
```

Comment:
Nit: "UIAccessibility" instead of "Accessibility"

Reply:
Done